#! /usr/bin/env python
# Forearm camera configuration

PACKAGE='rt_road_detection'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("width_cell", int_t, 0, "width cell", 16, 16, 64)
gen.add("width_block", int_t, 0, "width block", 64, 32, 128)
gen.add("height_block", int_t, 0, "height block", 64, 32, 128)
gen.add("height_cell", int_t, 0, "height cell", 16, 16, 64)
gen.add("svm_threshold", double_t, 0, "svm threshold", 0, -10.0, 10.0)
gen.add("flat_surface_in_block", double_t, 0, "flat surface in block", 0.5, 0, 1.0)


exit(gen.generate(PACKAGE, "lbp_detector_node", "LBPDetector"))
